### PR TITLE
Add new changefeed DB Console graphs

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-8058

This PR updates the Changefeed Dashboard page within the DB Console docs to add new graphs to the dashboard.

Also includes edits to existing metrics/graphs.

Rendered preview here: [src/current/v23.2/ui-cdc-dashboard.md](https://deploy-preview-18011--cockroachdb-docs.netlify.app/docs/v23.2/ui-cdc-dashboard.html)

### CODE REVIEW

**Comment by kathancox** [comment]
cc @amruss

**Review by miretskiy — APPROVED** [review]

File: src/current/v23.2/ui-cdc-dashboard.md [hunk 1]
```diff
@@ -9,65 +9,133 @@ The **Changefeeds** dashboard in the DB Console lets you monitor the [changefeed
...
 
-- The time in milliseconds per second required by CockroachDB to send messages to the sink.
-- The time CockroachDB spent waiting for the sink to flush the messages for changefeeds.
+<img src="{{ 'images/v23.2/ui-max-checkpoint-latency.png' | relative_url }}" alt="DB Console Max Checkpoint Latency graph" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/ui_sink_timings.png' | relative_url }}" alt="DB Console Sink Timings graph" style="border:1px solid #eee;max-width:100%" />
+{{site.data.alerts.callout_info}}
+The maximum checkpoint latency is distinct from, and slower than, the commit-to-emit latency for individual change messages. For more information about resolved timestamps, see [Ordering guarantees]({% link {{ page.version.version }}/changefeed-messages.md %}#ordering-guarantees).
```
Comment (florence-crl) [inline src/current/v23.2/ui-cdc-dashboard.md:77]: Is "commit-to-emit latency" the same as "commit latency" in the above graph? If so, for consistency, maybe use "commit latency".

File: src/current/v23.2/ui-cdc-dashboard.md [hunk 2]
```diff
@@ -9,65 +9,133 @@ The **Changefeeds** dashboard in the DB Console lets you monitor the [changefeed
...
-**Messages** | The number of messages that CockroachDB sent to the sink for changefeeds.
-**Flushes** | The the number of flushes that the sink performed for changefeeds.
+**Messages** | The number of messages that CockroachDB sent to the downstream sink.
+**Flushes** | The total number of flushes to the sink.
 
-## Sink Timings
+## Max Checkpoint Latency
 
-This graph shows:
+This graph displays the most any changefeed's persisted [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) is behind the present time. Larger values indicate issues with successfully ingesting or emitting changes. If errors cause a changefeed to restart, or the changefeed is [paused]({% link {{ page.version.version }}/pause-job.md %}) and unpaused, emitted data up to the last checkpoint may be re-emitted.
```
Comment (florence-crl) [inline src/current/v23.2/ui-cdc-dashboard.md:72]:
```suggestion
This graph displays the maximum time by which any changefeed's persisted [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) is behind the present time. Larger values indicate issues with successfully ingesting or emitting changes. If errors cause a changefeed to restart, or the changefeed is [paused]({% link {{ page.version.version }}/pause-job.md %}) and unpaused, emitted data up to the last checkpoint may be re-emitted.
```

**Review by florence-crl — COMMENTED** [review]
(a) 2 grammar suggestions
(b) The following 3 graph images are not updated for 23.2 (the attached images are from my v23.2-alpha.5 cluster, however they have no non-zero data)
- Changefeed Status
- Commit Latency
- Rangefeed catchup scans duration 
Screenshot 2023-11-01 at 1 51 50 PM
Screenshot 2023-11-01 at 1 52 13 PM
Screenshot 2023-11-01 at 1 52 40 PM

**Comment by kathancox** [comment]
> (a) 2 grammar suggestions (b) The following 3 graph images are not updated for 23.2 (the attached images are from my v23.2-alpha.5 cluster, however they have no non-zero data)
> 
> * Changefeed Status
> * Commit Latency
> * Rangefeed catchup scans duration
> 
> Screenshot 2023-11-01 at 1 51 50 PM Screenshot 2023-11-01 at 1 52 13 PM Screenshot 2023-11-01 at 1 52 40 PM

Hey @florence-crl, thanks for the review! Regarding this comment, I'm not sure of your intention re the screenshots?

**Comment by kathancox** [comment]
@florence-crl Yes cool, see "Edit" in my comment, how would you like me to update the nav include file? I am happy to do that in this PR — I just want to make sure I'm covering what's needed and you have more insight into this area.

**Comment by florence-crl** [comment]
Please change the following line in `_includes/v23.2/ui/ui-metrics-navigation.md`
from

> When viewing graphs, a tooltip will appear at your mouse cursor providing further insight into the data under the mouse cursor. Click anywhere within the graph to pin the tooltip in place, decoupling the tooltip from your mouse movements. Click anywhere within the graph to cause the tooltip to follow your mouse once more.

to

> When viewing graphs, two perpendicular lines will appear at your mouse cursor providing further insight into the data under the mouse cursor. The metric values are displayed in the legend under the graph. Click anywhere within the graph to pin the values in place, decoupling the value changes from your mouse movements. Click anywhere within the graph to cause the values to change with your mouse movements once more.

Please edit the line as needed.

**Review by florence-crl — APPROVED** [review]
LGTM!
1 thing was unclear, did you see [this suggestion](https://github.com/cockroachdb/docs/pull/18011#discussion_r1379120592)? it was not marked resolved.

**Comment by kathancox** [comment]
> LGTM! 1 thing was unclear, did you see [this suggestion](https://github.com/cockroachdb/docs/pull/18011#discussion_r1379120592)? it was not marked resolved.

@florence-crl thanks, yes all suggestions included. Thank you!

**Pull request merged**